migrate_spoken/views.py

```diff
@@ -1,33 +1,37 @@
 from django.shortcuts import render
 from django.http import HttpResponse, HttpResponseRedirect
-#from website.models import Question, Answer, AnswerComment, Notification
+# from website.models import Question, Answer, AnswerComment, Notification
 from migrate_spoken.models import OldUsers
 from spoken_auth.models import Users
 # Create your views here.
 
+
 def get_old_user(uid):
     try:
-        return OldUsers.objects.get(pk = uid)
+        return OldUsers.objects.get(pk=uid)
     except Exception, e:
         return None
 
+
 def get_current_user_from_old_email(mail):
     try:
-        return Users.objects.get(email = mail)
+        return Users.objects.get(email=mail)
     except Exception, e:
         print e
         return None
 
+
 def get_current_user_id_from_old_uerid(uid):
     old_user = get_old_user(uid)
     if old_user:
-        #print old_user.uid, " => ", old_user.mail
+        # print old_user.uid, " => ", old_user.mail
```

Comment:
Print statements are unnecessary, maintaining commented code containing print statements is equally if not more unnecessary.
These are the sins of those who coded before you. You have been bestowed the task of doing the right thing by fixing their wrongs.
It is a difficult mission and I hope you will do the right thing